opencog/atoms/base/Handle.cc

```diff
@@ -80,6 +80,21 @@ bool Handle::atoms_less(const Atom* pa, const Atom* pb)
     return a < b;
 }
 
+bool Handle::content_based_atoms_less(const Atom* a, const Atom* b)
+{
+    if (a == b) return false;
+    if (nullptr == a) return true;
+    if (nullptr == b) return false;
+    UUID ua = a->getUUID();
+    UUID ub = b->getUUID();
```

Comment:
Ah, OK, sorry.  I probably wrote that code, and so shouldn't be so touchy. The meta-issue is that I seem to spend too much time struggling with pointless stupid bugs, and not enough time advancing the high theory and important functions.  Its like, the rocket can't launch because a bolt fell out.  So I get snappy when I see anything resembling a loose bolt.